src/Symfony/Component/PropertyInfo/Tests/Fixtures/ConstructorDummy.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Symfony\Component\PropertyInfo\Tests\Fixtures;
+
+/**
+ * @author Dmitrii Poddubnyi <dpoddubny@gmail.com>
+ */
+class ConstructorDummy
+{
+    /** @var string */
+    private $timezone;
+
+    /** @var \DateTimeInterface */
+    private $date;
+
+    /** @var int */
+    private $dateTime;
+
+    /**
+     * @param \DateTimeZone      $timezone
+     * @param int                $date       Timestamp
+     * @param \DateTimeInterface $dateObject
+     */
+    public function __construct(\DateTimeZone $timezone, $date, $dateObject, \DateTime $dateTime)
```

Comment:
```suggestion
    public function __construct(\DateTimeZone $timezone, int $date, \DateTimeInterface $dateObject, \DateTime $dateTime)
```

Reply:
but `$dateObject` isn't used´ .... 🤔

Reply:
@OskarStark  That's on purpose, they are used for testing phpdoc types extraction from constructor even if no such property exists.